ramls/patron-block-conditions.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: Patron Block Conditions
+version: v0.1
+baseUri: http://github.com/org/folio/mod-users
+
+documentation:
+  - title: mod-users Patron Block Conditions API
+    content: This documents the API calls that can be made to query and manage patron blocks conditions
+
+types:
+  patron-block-condition: !include patron-block-condition.json
+  patron-block-conditions: !include patron-block-conditions.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  pageable: !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  language: !include raml-util/traits/language.raml
+  validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/patron-block-conditions:
```

Comment:
Can you add the new API to https://github.com/folio-org/mod-users/blob/master/descriptors/ModuleDescriptor-template.json ?

Reply:
added, thanks